python/fileaway.py

```diff
@@ -178,11 +180,11 @@ def check_timer():
 
 if __name__ == "__main__":
   if w.register(SCRIPT_NAME, SCRIPT_AUTHOR, SCRIPT_VERSION, SCRIPT_LICENSE, SCRIPT_DESC, "", ""):
-    for option, default_value in settings.iteritems():
+    for option, default_value in iter(settings.items()):
```

Comment:
```suggestion
    for option, default_value in settings.items():
```
Using `.items()` is enough here.